frontend/src/components/ConfirmDeletionModal.svelte

```diff
@@ -16,23 +17,29 @@
     }
 
     function deleteObject() {
-        //TODO
+        fetch("/api" + to_delete, {
+            method: 'DELETE',
+        })
+            .then((response) => response.json())
+            .then((response_data) => (response_data))
+            .catch(err => console.log(err))
+        window.location.href = "/categories"
```

Comment:
[Use goto() for internal navigation](https://kit.svelte.dev/docs/modules#$app-navigation-goto)